Update Actuator.cpp

fbusche opened the pull request:
just framework, much needed still. variables need to be changes from set constants to the actual input gotten from the joystick. need to also see about running this in loop.

fbusche committed:
Update Actuator.cpp

just framework, much needed still. variables need to be changes from set constants to the actual input gotten from the joystick. need to also see about running this in loop.

Tpanda03 commented:
Errors came from the class containing a main, refactor functions to match class definitions in .h file

fbusche committed:
Update Actuator.cpp

still need to know what to do with values gotten for each actuator so that we can actually move them that amount. however, in this the framework for how the class is set up has been changed to using methods rather than cases. Additionally, the setters and getters were better set up and the main() function that caused an error in the first place was removed entirely.

fbusche committed:
Update Actuator.cpp

changed to a tuple for the x_scale, y_scale, and z_scale values. also removed the setters and getters for the joystickOutput array and am just changing it so that it's a class member vairable in the .hpp file

fbusche committed:
Update Actuator.hpp

added the joystickOutput[] variable as a private class member.
